File: source/utils/heroku.py
from io import open
from os import path


def heroku():
    heroku_file = 'heroku.yml'

    heroku_file_content = '''
build:
  docker:
    web: Dockerfile
'''

    with open(heroku_file, 'w') as f:
        f.write(heroku_file_content)

File: lib/src/functions/heroku.js
const writeFile = require('../utils/writeFile')

module.exports = async () => {
  const data = {
    herokuContent: `build:
  docker:
    web: Dockerfile`,
    herokuFile: 'heroku.yml'
  }

  await writeFile(data.herokuFile, data.herokuContent)
}
